bin/update-database

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -euo pipefail
+
+base="$(dirname "$0")/.."
+
+main() {
+    local records="$1"
+    local database="$2"
+    local table=record_barcodes
+
+    # XXX TODO: This whole block could be a new --truncate option to
+    # `sqlite-utils insert` once that's available.  That'd be nicer because
+    # it'd group the delete and insert into the same transaction, leaving no
+    # room for an empty table after a successful delete but failed insert.
+    #   -trs, 7 July 2020
+    if [[ -e "$database" ]]; then
+        # sqlite3 appears to exits non-zero on a SQL error only when the SQL
+        # statement is given as an argument; when reading from stdin it always
+        # seems to exit 0.
+        sqlite3 "$database" "delete from $table"
+    fi
```

Comment:
Did you like the idea of explicitly doing a SELECT COUNT(*) FROM ... to ensure that the delete worked and then failing if not?

Reply:
Oh, right. I think that check somehow got lost from our pair programming session.

Reply:
I think the comments above explain the lack of the special `select` test? 
Since the SQL is provided as an argument, sqlite3 will exit with error.